packages/opentelemetry/lib/index.js

```diff
@@ -0,0 +1,89 @@
+const { diag, DiagLogLevel } = require('@opentelemetry/api');
+const {
+	getNodeAutoInstrumentations
+} = require('@opentelemetry/auto-instrumentations-node');
+const { Resource } = require('@opentelemetry/resources');
+const opentelemetrySDK = require('@opentelemetry/sdk-node');
+const {
+	SemanticResourceAttributes
+} = require('@opentelemetry/semantic-conventions');
+const appInfo = require('@dotcom-reliability-kit/app-info');
+const {
+	OTLPTraceExporter
+} = require('@opentelemetry/exporter-trace-otlp-proto');
+const { NoopSpanProcessor } = require('@opentelemetry/sdk-trace-base');
+const logger = require('@dotcom-reliability-kit/logger');
+
+/**
+ * Set up OpenTelemetry tracing.
+ *
+ * @param {object} options
+ *      Options passed into the function parameter.
+ * @param {string | null} [options.tracesEndpoint]
```

Comment:
This line is oddly failing linting as it's saying:

```
/Users/cynthia.mbulu/Documents/code/dotcom-reliability-kit/packages/opentelemetry/lib/index.js
  22:1  error  Sentences should start with an uppercase character  jsdoc/require-description-complete-sentence
```

but this linting error doesn't make sense here 😭

Reply:
Oh it's the JSDoc plugin being a pain. it sees the `.` in `e.g.` as the end of a sentence 🤦 maybe rephrase as:

> The URL to send OpenTelemetry trace segments to, for example http://localhost:4318/v1/traces.

Reply:
OMG 😅  - it's so nitpicky. Thank you!!